packages/loader/container-loader/src/test/serializedStateManager.spec.ts

```diff
@@ -175,7 +167,32 @@ describe("serializedStateManager", () => {
 		);
 	});
 
-	it("can get snapshot from previous local state", async () => {
+	it("can get pending local state after attach", async () => {
+		const storageAdapter = new MockStorageAdapter();
+		const serializedStateManager = new SerializedStateManager(
+			undefined,
+			logger,
+			storageAdapter,
```

Comment:
do we expect any calls to be made on the storage adapter is this case? if not, using something like the failProxy from packages\loader\container-loader\src\test\loader.spec.ts is a good way to validate that